test/CMakeLists.txt.in

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.4)
```

Comment:
Are we making this mandatory? What if one already has Gtest on its system?

Reply:
This is the same way that the Firmware does it, it means you aren't dependent on system setup as much.